python/pyarrow/table.pxi

```diff
@@ -163,7 +163,16 @@ cdef class ChunkedArray(_PandasConvertible):
                 self.sp_chunked_array,
                 self, &out))
 
-        return pandas_api.series(wrap_array_output(out), name=self._name)
+        result = pandas_api.series(wrap_array_output(out), name=self._name)
+
+        if isinstance(self.type, TimestampType):
+            tz = self.type.tz
+            if tz is not None:
+                tz = string_to_tzinfo(tz)
+                result = (result.dt.tz_localize('utc')
+                          .dt.tz_convert(tz))
```

Comment:
I am not sure those 6 lines are worth a separate helper function (now this is duplicated from Array.to_pandas), but if others prefer I can certainly do that

Reply:
Seems fine to me, it can be deduplicated in the future if needed. @wesm or @pitrou any preference?

Reply:
It's small but carries domain knowledge, so I'd rather see a helper function (perhaps in the `pandas_compat` module).

Reply:
OK, I made a small helper function (see new push). 
For now I still put it behind the if check to only need to import it from pandas_compat if it would actually be needed. But that made that it is not much shorter as before in lines of code .. If find this clearer in the code to read, but I can also always call the helper function and move the if check inside the helper function.